doc/Guides/Authentication.md

```diff
@@ -49,6 +49,7 @@ To make your own custom Authenticator, you can just create a new script in your
 Here are some tips for custom Authenticators:
 
 -   `OnStartServer` and `OnStartClient` are the appropriate methods to register server and client messages and their handlers. They're called from StartServer/StartHost, and StartClient, respectively.
+-   The base Authenticor includes a timeout feature so you can prevent rogue clients from locking up your connections.
```

Comment:
Typo: "Authenticor" -> "Authenticator"
This is a nice feature!

Reply:
Fixed